lib/services/apigateway/index.js

```diff
@@ -83,6 +83,16 @@ function getCompiledApiGatewayTemplate(stackName, ownServiceContext, dependencie
         tags: deployPhaseCommon.getTags(ownServiceContext)
     }
 
+		// Add binary media types if specified
+		if(serviceParams.binary_media_types)
+    {
+      handlebarsParams.binaryMediaTypes=[];
+      for(let type of serviceParams.binary_media_types)
```

Comment:
This won't work when the user hasn't specified binary_media_types, because serviceParams.binary_media_types will be undefined, which isn't iterable, so we should probably put this in an if statement checking if the user has defined binary_media_types.